### ISSUE TITLE
Unexpected Error

### ISSUE THREAD

**Issue by cgroveacademy** [OpenLiveWriter/OpenLiveWriter#300]
I posted a draft to my blog today, then I went back in to edit it in OLW and went to schedule it...  I got this The service blogger has thrown an exception: Google.GoogleApiException: Google.Apis.Requests.RequestError
Not Found [404]
Errors [
	Message[Not Found] Location[ - ] Reason[notFound] Domain[global]
]

   at Google.Apis.Requests.ClientServiceRequest`1.Execute()
   at OpenLiveWriter.BlogClient.Clients.GoogleBloggerv3Client.EditPost(String blogId, BlogPost post, INewCategoryContext newCategoryContext, Boolean publish, String& etag, XmlDocument& remotePost)
   at OpenLiveWriter.BlogClient.Blog.EditPost(BlogPost post, INewCategoryContext newCategoryContext, Boolean publish)
   at OpenLiveWriter.PostEditor.UpdateWeblogAsyncOperation.DoWork()
   at OpenLiveWriter.CoreServices.AsyncOperation.InternalStart()

I deleted it from blogger (the draft) and tried to schedule the post again, same thing.  Any ideas?  

And now I can't even make a new post... it says 
The remote server returned an error: (500) Internal Server Error.

**Comment by ScottIsAFool** [comment]
I believe the 500 internal server error should have stopped happening. Are you still getting the original error you reported?

**Comment by cgroveacademy** [comment]
the 500 did stop, thank you.
I ended up just cutting and pasting the post directly into blogger, but even after the 500 error stopped, I was still having issues.  But I just tried it and it worked!  Thank you!   Jen Altman

Usborne Books, Close to My Heart & dOTERRA

www.chestnutgroveacademy.blogspot.com
www.ubah.com/L2096www.myctmh.com/jenaltmanwww.mydoterra.com/chestnutgroveacademy

 

    On Thursday, January 7, 2016 8:59 AM, Scott Lovegrove <notifications@github.com> wrote:
 

 I believe the 500 internal server error should have stopped happening. Are you still getting the original error you reported?—
